### ISSUE TITLE
Editor content box: incorrect sizing

### ISSUE THREAD

**Issue by mikerockett** [NicoKnoll/LightWire-Skin#1]
Seems that something's gone terribly wrong here:

![lightwireckeditor](https://cloud.githubusercontent.com/assets/4586280/6212825/e33deade-b5f1-11e4-8751-17a3c38fb6a6.png)

Only once was the width correct... Upon refresh, it broke again... And no errors in the console.

If I resize the browser, it then goes to the correct size.

Using Opera 27.

**Comment by mikerockett** [comment]
Also, the the editor won't even appear in Firefox... Does seem to load-up (markup is there), but the entire editor is invisible.

**Comment by nicoknoll** [comment]
Fixed the first one already but haven't pushed it so far.

**Comment by mikerockett** [comment]
Ah, great - thanks. :+1:

**Comment by mikerockett** [comment]
Last one: the icons don't render so great on Windows, as you can see in the screenie. I guess it's the only thing about Windows I can't stand is the mess-up with font-rendering. In the console, I sized the font-size for the buttons up to 15px, but it looked bloated (and not light, as the name implies). I guess one would have to change the colour (to grey, for example) to get it out of the way...

Alternatively: SVG

(Should I open up a new issue for this?)

**Comment by nicoknoll** [comment]
Well I use fontawesome to match the PW icons. So rendering should be as good as the other PW Icons.

Fixed the firefox issue btw.

**Comment by mikerockett** [comment]
Well, they do normally render okay, but not at 12px.

Can I suggest the following changes?
1. `.cke_button`s padding should change to `5px 8px 6px`
2. `.cke_button_icon:before`s font size should change from 12px to 14px
3. `.cke_button_icon`s margin-top should be changed from 1px to 0

On Windows, the result is this (before and after):

![lightwireckeditor_changes](https://cloud.githubusercontent.com/assets/4586280/6213828/792ec1c6-b5fc-11e4-940c-97cd2a1d98cf.png)

**Update:** To retain height, we can add a `margin-bottom: 1px` to `cke_button_icon`. Looks perfect on my end :+1:

**Comment by nicoknoll** [comment]
Sounds good to me. I'll try it :)

**Comment by mikerockett** [comment]
Should look good on Mac too. Everything looks good on Mac. :wink:

**Comment by nicoknoll** [comment]
Better?

**Comment by mikerockett** [comment]
Perfect :+1:
